tests/Fake_data_generator_test.py

```diff
@@ -0,0 +1,62 @@
+'''
```

Comment:
Similarly to the MultiChannelReaderProcessor, it would be good to merge this unit test with the TritiumSpectrum one...

Reply:
If you don't, make sure to also add this test script into the test.sh file

Reply:
I added it to test.sh